.coderabbit.yaml

```
@@ -0,0 +1,165 @@
+tone_instructions: ''
+early_access: false
+enable_free_tier: true
+reviews:
+  profile: chill
+  request_changes_workflow: false
+  high_level_summary: true
+  high_level_summary_placeholder: '@coderabbitai summary'
+  high_level_summary_in_walkthrough: false
+  auto_title_placeholder: '@coderabbitai'
+  auto_title_instructions: |
+    Generate a pull request title following the Conventional Commits specification. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)